apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ServersSummaryView.kt

```diff
@@ -0,0 +1,992 @@
+      variableValue = statusColorAndPercentage.variableValue)
+    if (pref.state.value && !leadingPercentage) SubscriptionStatusIndicatorPercentage(percentageText)
+  }
+}
+
+enum class PresentedUserCategory {
+  CURRENT_USER, ALL_USERS
+}
+
+enum class PresentedServerType {
+  SMP, XFTP
+}
+
+@Composable
+private fun ServerSessionsView(sess: ServerSessions) {
+  SectionView(generalGetString(MR.strings.servers_info_transport_sessions_section_header).uppercase()) {
+    InfoRow(
+      generalGetString(MR.strings.servers_info_sessions_connected),
+      numOrDash(sess.ssConnected)
+    )
+    InfoRow(
+      generalGetString(MR.strings.servers_info_sessions_errors),
+      numOrDash(sess.ssErrors)
+    )
+    InfoRow(
+      generalGetString(MR.strings.servers_info_sessions_connecting),
+      numOrDash(sess.ssConnecting)
+    )
+  }
+}
+
+private fun serverAddress(server: String): String {
+  val address =  parseServerAddress(server)
+
+  return address?.hostnames?.first() ?: server
+}
+
+@Composable
+private fun SMPServerView(srvSumm: SMPServerSummary, statsStartedAt: Instant, rh: RemoteHostInfo?) {
+  SectionItemView(
+    click = {
+      ModalManager.start.showCustomModal { close ->
+        SMPServerSummaryView(
+          rh = rh,
+          close = close,
+          summary = srvSumm,
+          statsStartedAt = statsStartedAt
+        )
+      }
+    }
+  ) {
+    Text(
+      serverAddress(srvSumm.smpServer),
+      modifier = Modifier.weight(10f, fill = true)
+    )
+    if (srvSumm.subs != null) {
+      Spacer(Modifier.fillMaxWidth().weight(1f))
+      SubscriptionStatusIndicatorView(subs = srvSumm.subs, sess = srvSumm.sessionsOrNew, leadingPercentage = true)
+    } else if (srvSumm.sessions != null) {
+      Spacer(Modifier.fillMaxWidth().weight(1f))
+      Icon(painterResource(MR.images.ic_arrow_upward), contentDescription = null, tint = SessIconColor(srvSumm.sessions))
+    }
+  }
+}
+
+@Composable
+private fun SessIconColor(sess: ServerSessions): Color {
+  val online = chatModel.networkInfo.value.online
+  return if (online && sess.ssConnected > 0) SessionActiveColor() else MaterialTheme.colors.secondary
+}
+
+@Composable
+private fun SessionActiveColor(): Color {
+  val netCfg = remember { chatModel.controller.getNetCfg() }
+  val socksProxy = remember { netCfg.socksProxy }
+  return if (socksProxy != null) Indigo else MaterialTheme.colors.primary
+}
+
+@Composable
+private fun SMPServersListView(servers: List<SMPServerSummary>, statsStartedAt: Instant, header: String? = null, footer: String? = null, rh: RemoteHostInfo?) {
+  val sortedServers = servers.sortedWith(compareBy<SMPServerSummary> { !it.hasSubs }
+    .thenBy { serverAddress(it.smpServer) })
+
+  SectionView(header) {
+    sortedServers.map { svr -> SMPServerView(srvSumm = svr, statsStartedAt = statsStartedAt, rh = rh) }
+  }
+  if (footer != null) {
+    SectionTextFooter(
+      footer
+    )
+  }
+}
+
+fun prettySize(sizeInKB: Long): String {
```

Comment:
could refactor this with `formatBytes` from Util, but that one prints with zeros after dot (e.g. "123.00 MB"), so I kept both. maybe we should reuse this logic for `formatBytes` instead